### ISSUE TITLE
error on Mac OS

### ISSUE THREAD

**Issue by iamvicliu** [leolewis/openvisualtraceroute#16]
I have installed java

————————
ERROR launching 'OpenVisualTraceroute'.

No suitable Java version found on your system!
This program requires Java 17.0.1 or later.
Make sure you install the required Java version.

**Comment by spacerunner5** [comment]
Same here, macOS 10.15.7.  Updating Java did not help.

After installing [Java SE](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html) it worked.

**Comment by iloving** [comment]
I just ran into this now.  I have been trying my absolute best to avoid anything Oracle because of their abusive policies, so it is EXTREMELY frustrating that an open source tools of all things is forcing me to install a proprietary JDK.

Edit:  If you download one of the nightly builds of AdoptJDK 17, it appears to be working.  I haven't tested it exhaustively yet but  at least it runs.  https://adoptium.net/temurin/nightly/

**Comment by hoegge** [comment]
Yes, so the link in the dialog box should really be to the Java SE download page and not to the "Java 8" page
